lib/friendly_id/sequential_slug_generator.rb

```diff
@@ -0,0 +1,53 @@
+module FriendlyId
+  class SequentialSlugGenerator
+    attr_accessor :scope, :slug, :slug_column, :sequence_separator
+
+    def initialize(scope, slug, slug_column, sequence_separator)
+      @scope = scope
+      @slug = slug
+      @slug_column = slug_column
+      @sequence_separator = sequence_separator
+    end
+
+    def generate
+      slug + sequence_separator + next_sequence_number.to_s
```

Comment:
It is not safe to just use '+' operator. In some of my examples scope can be nil and this method throws an error. You could return nil if scope/slug is nil or raise a custom error (just an idea).